core/app/finders/spree/taxons/find.rb

```diff
@@ -0,0 +1,52 @@
+module Spree
+  module Taxons
+    class Find
+      def initialize(taxons, params)
+        @taxons = taxons
+        @params = params
+      end
+
+      def execute
+        taxons = @taxons
```

Comment:
@bbonislawski 
This re-assignment doesn't make a lot of sense to me. We already got taxons in instance variable and we can use `attr_reader` to access it properly. Why would we re-assign it? That's basically useless memory allocation at this point